Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
No docs of what it does. Please explain it here with info on how it affects the compile time.